package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "theia-sprotty",
-  "version": "0.1.5",
+  "version": "0.1.6",
```

Comment:
Why did you increase the version number? This is usually done by npm when publishing.